cmd/keygenerator/main.go

```diff
@@ -116,10 +120,12 @@ VERSION:
 
 	walletKeyFilenameTemplate    = "walletKey%s.pem"
 	validatorKeyFilenameTemplate = "validatorKey%s.pem"
+	p2pKeyFilenameTemplate       = "p2pKey%s.pem"
 
 	log = logger.GetOrCreate("keygenerator")
 
 	validatorPubKeyConverter, _ = pubkeyConverter.NewHexPubkeyConverter(blsPubkeyLen)
+	p2pPubKeyConverter, _       = pubkeyConverter.NewHexPubkeyConverter(blsPubkeyLen)
```

Comment:
let's not use the hex pub key converter here as we might want to see in the pem file something like:
```
-----BEGIN PRIVATE KEY for 16Uiu2HAm7deAWdBU41D6SUgtUEKC2VpXuRay4QqWnnPLqa8358Yv-----
```
https://github.com/ElrondNetwork/elrond-go/blob/5aed6c6ac41ded6321323a5195a03e6329fa50d0/p2p/libp2p/p2pSigner.go#L23
suggestion to check how is it done in the

Reply:
added a separate converter for p2p encoding